openmc/statepoint.py

```diff
@@ -523,9 +524,18 @@ def get_tally(self, scores=[], filters=[], nuclides=[],
             The id specified for the Tally (default is None).
         estimator: str, optional
             The type of estimator ('tracklength', 'analog'; default is None).
-        exact : bool
-            Whether to strictly enforce the match between the parameters and
-            the returned tally
+        exact_filters : bool
+            If True, the number of filters in the parameters must be identical
+            to those in the matching Tally. If False (default), the filters in
+            the parameters may be a subset of those in the matching Tally.
+        exact_nuclides : bool
+            If True, the number of nuclides in the parameters must be identical
+            to those in the matching Tally. If False (default), the nuclides in
+            the parameters may be a subset of those in the matching Tally.
+        exact_scores : bool
+            If True, the number of scores in the parameters must be identical
+            to those in the matching Tally. If False (default), the scores
+            in the parameters may be a subset of those in the matching Tally.
```

Comment:
The parameters in the method declaration are ordered `exact_nuclides`, `exact_filters`, `exact_scores` while the ordering here is `exact_filters`, `exact_nuclides`, `exact_scores`. Is consistency in the ordering of the parameters in the method declaration and docstring comment a standard that should be maintained? Looking at other files it appears so.